src/System.Threading.Overlapped/src/System.Threading.Overlapped.builds

```diff
@@ -10,11 +10,7 @@
     </Project>
     <Project Include="System.Threading.Overlapped.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>net46</TargetGroup>
-    </Project>
-    <Project Include="System.Threading.Overlapped.csproj">
```

Comment:
We should only remove it if we have shipped it in a stable package so we can harvest it from there. I think that is the case here so it should be OK but I wanted to call it out because we have removed some configurations recently (namely System.Console) where that wasn't the case.